dnd-bot/dnd_bot/logic/prototype/items/item.py

```diff
@@ -38,10 +40,25 @@ def load_attributes_from_json(self):
 
             for item_type in items.keys():
                 if self.name in items[item_type]:  # items without a subtype
-                    pass
+                    print(f"{self.name} is without a subtype")
```

Comment:
należałoby wyrzucić jakiś wyjątek a nie printować, kto na konsole będzie patrzył na produkcji

Reply:
to wcale nie musi być błąd, raczej informacja

Reply:
do usunięcia